### ISSUE TITLE
is it possible to compile .a android library with lime?

### ISSUE THREAD

**Issue by Sirox228** [openfl/lime#1588]
i just realized, that if i use .a libs, for some reason performance of the app is better, thats why i'm asking for this.
So is there any build flag for it? if answer is no, then, i think that i should make a feature request or smth?

P.S. if you have questions about how i found out that .a have better performance than .so, if lime can't do so, then: i'm using linc_luajit in my app and i compiled luajit libs using make with my hands and suddenly found out that performance became better

**Comment by player-03** [comment]
Did you try compiling a .so file using Make?

**Comment by Sirox228** [comment]
yes, make compiling for both .so and .a at the same time and you can find both libs after running single build command
